Makefile

```diff
@@ -400,7 +406,7 @@ run-k8s-apiserver: stop-k8s-apiserver run-etcd
 	# Create CustomResourceDefinition (CRD) for Calico resources
 	# from the manifest crds.yaml
 	while ! docker exec st-apiserver kubectl \
-		apply -f /manifests/crds.yaml; \
+		apply -f https://raw.githubusercontent.com/projectcalico/libcalico-go/241d8b0486a3c732e938cfe6a7937e65e1bad319/test/crds.yaml; \
```

Comment:
Can we avoid pinning this to a specific commit using `go list ...` magic?

Reply:
Fixed

Reply:
Fixed

Reply:
This one is a bit more tricky and I couldn't get it to work with `go list ...`. Let's follow up on this later.

Reply:
Fixed